cmd/vela-worker/operate.go

```diff
@@ -23,6 +26,38 @@ func (w *Worker) operate() error {
 	if err != nil {
 		return err
 	}
+	// Define the database representation of the worker
+	// and register itself in the database
+	workerActive := true
+	workerLastCheckedIn := time.Now().Unix()
+	registryWorker := library.Worker{
+		Hostname:      &w.Config.Hostname,
+		Address:       &w.Config.Server.Address,
+		Routes:        &w.Config.Queue.Routes,
+		Active:        &workerActive,
+		LastCheckedIn: &workerLastCheckedIn,
+	}
+	w.register(&registryWorker)
+	if err != nil {
+		logrus.Error("unable to register worker with the server")
+	}
+
+	// spawn goroutine for phoning home
+	go func() {
+		for {
+			// sleep for the configured time
+			time.Sleep(*w.Config.CheckIn)
+
+			// set checking time to now and call the server
+			registryWorker.SetLastCheckedIn(time.Now().Unix())
```

Comment:
We'll want to make sure that we're using the `UTC` format for the timestamps 👍 

```suggestion
			registryWorker.SetLastCheckedIn(time.Now().UTC().Unix())
```